scifight/models.py

```diff
@@ -206,8 +234,9 @@ class Meta:
 
 
 class LeaderToJury(models.Model):
-    leader = models.ForeignKey(Leader)
-    jury = models.ForeignKey(Jury)
+    tournament  = models.ForeignKey(Tournament)
+    leader      = models.ForeignKey(Leader)
+    jury        = models.ForeignKey(Jury)
```

Comment:
Okay for now, but we're going to replace the entire `LeaderToJury` model with `HumanEntity` soon.